packages/cli/dependencies.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# all colors: https://en.wikipedia.org/wiki/ANSI_escape_code
+NOCOLOR='\033[0m'
+YELLOW='\033[0;33m'
+GRAY='\033[0;37m'
+GREEN='\033[0;32m'
+CYAN='\033[0;36m'
+
+if [ -f "/etc/odbcinst.ini" ]; then
+    echo "Dependency ODBC exists"
```

Comment:
btw this will differ based on OS/package manager

<img width="486" alt="CleanShot 2022-10-04 at 18 58 18@2x" src="https://user-images.githubusercontent.com/962095/193853797-9410a6fe-9ee8-4507-b4a3-d36d8a0018bd.png">

I have odbc on mac installed using brew
my odbcinst path is `/opt/homebrew/bin/odbcinst`